From ff03f0bd7dcb6180c22713b958d32c777752c6d2 Mon Sep 17 00:00:00 2001
From: Josh Triplett <josh@joshtriplett.org>
Date: Sun, 22 Apr 2012 20:26:42 -0700
Subject: [PATCH 4/4] Update Recommends, Suggests, and package description.

* Update Recommends and Suggests:
  * Recommends at least one of a syslog daemon or a mail transport agent, to
    ensure that the recommended configuration will capture cron job output
    somewhere.
  * Separately, suggest a syslog daemon and suggest a mail transport agent.
  * Use default-mta rather than hard-coding specific MTAs.
* Update package description to document the requirement for either a syslog
  daemon or a mail transport agent to capture cron job output.
---
 debian/changelog |    8 ++++++++
 debian/control   |   10 ++++++----
 2 files changed, 14 insertions(+), 4 deletions(-)

diff --git a/debian/changelog b/debian/changelog
index d33c247..e500460 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -12,6 +12,14 @@ cron (3.0pl1-122) UNRELEASED; urgency=low
     cron job output when not already mailing that output.
   * In /etc/default/cron, change the default log level to -L 35, to ensure
     that cron job output gets logged on systems without an MTA.
+  * Update Recommends and Suggests:
+    * Recommends at least one of a syslog daemon or a mail transport agent, to
+      ensure that the recommended configuration will capture cron job output
+      somewhere.
+    * Separately, suggest a syslog daemon and suggest a mail transport agent.
+    * Use default-mta rather than hard-coding specific MTAs.
+  * Update package description to document the requirement for either a syslog
+    daemon or a mail transport agent to capture cron job output.
 
  -- Javier Fernández-Sanguino Peña <jfs@debian.org>  Mon, 09 Apr 2012 17:23:01 +0200
 
diff --git a/debian/control b/debian/control
index 89ea77a..6f178a1 100644
--- a/debian/control
+++ b/debian/control
@@ -25,9 +25,11 @@ Depends:
     lsb-base (>= 3.0-10),
     libpam-runtime (>= 1.0.1-11)
 Recommends:
-    exim4 | postfix | mail-transport-agent
+    rsyslog | system-log-daemon | default-mta | mail-transport-agent
 Suggests:
     anacron (>=2.0-1),
+    default-mta | mail-transport-agent,
+    rsyslog | system-log-daemon,
     logrotate,
     checksecurity
 Breaks:
@@ -39,9 +41,9 @@ Description: process scheduling daemon
  specified in a crontab. By default, users may also create crontabs of
  their own so that processes are run on their behalf.
  .
- Output from the commands is usually mailed to the system administrator
- (or to the user in question); you should probably install a mail system
- as well so that you can receive these messages.
+ Output from cron jobs is either mailed to the user or logged to the system
+ log.  You should install a syslog daemon if you want to log cron job output, or
+ install a mail transport agent if you want to receive cron job output by email.
  .
  This cron package is configured by default to do some basic daily system
  maintenance tasks. Other packages, such as checksecurity, can provide further
-- 
1.7.10

